site/pages/core/authenticate-users.mdx

```diff
@@ -50,6 +50,38 @@ await signer.authenticate({ type: "email", bundle });
 
 :::
 
+## Social Login
+
+You can allow uers to sign in with a Oauth provider like Google, Facebook, or Auth0. There are two login flows that are possible: the redirect flow goes to the auth provider's login page and the popup flow has the auth provider login screen to open as a popup.
```

Comment:
Typo: "uers" -> "users".

Also, let's stylize "OAuth" rather than "Oauth" in written text.

Also, I feel that some of the phrasing is unclear in that in my mind both flows "go" to the auth provider's login page. I would possibly phrase it as something like "the redirect flow redirect's the user to the auth provider's login page, then back to your application when done. The popup flow opens the auth provider in a popup window, then updates your application when done." Since that's wordier, we might have bullet points for the two options.